src/Http/Cookie/SameSiteEnum.php

```diff
@@ -0,0 +1,68 @@
+<?php
+declare(strict_types=1);
+
+/**
+ * CakePHP(tm) : Rapid Development Framework (http://cakephp.org)
+ * Copyright (c) Cake Software Foundation, Inc. (http://cakefoundation.org)
+ *
+ * Licensed under The MIT License
+ * Redistributions of files must retain the above copyright notice.
+ *
+ * @copyright     Copyright (c) Cake Software Foundation, Inc. (http://cakefoundation.org)
+ * @link          http://cakephp.org CakePHP(tm) Project
+ * @since         5.0.0
+ * @license       http://www.opensource.org/licenses/mit-license.php MIT License
+ */
+namespace Cake\Http\Cookie;
+
+use ValueError;
+
+// phpcs:ignoreFile
+enum SameSiteEnum: string
+{
+    case Lax = 'Lax';
```

Comment:
Wondering if we should use uppercase for the names and keep capital case only for the values.

Reply:
Kind of does look like constants, doesnt it.

Reply:
I'm used to supper case enums where words are split by `_` - in other langauges  I don't know how often we'll have multi-word enums in cake.

Reply:
:+1: for upper case, harder to make derpy mistakes.